install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -518,14 +526,9 @@ conntrackGCInterval: ""
 containerRuntime:
   # -- Enables specific integrations for container runtimes.
   # Supported values:
-  # - containerd
   # - crio
-  # - docker
   # - none
-  # - auto (automatically detect the container runtime)
   integration: none
-  # -- Configure the path to the container runtime control socket.
-  # socketPath: /path/to/runtime.sock
```

Comment:
One last thing maybe, could you please add a comment above that block to specify that the value is deprecated (and also so we can remember to remove it)?

Something like:

```
# -- Deprecated in favor of bpf.autoMount.enabled. To be removed in 1.15.
```

Reply:
done

Reply:
Thanks a lot!